test/Microsoft.AspNetCore.TestHost.Tests/TestServerTests.cs

```diff
@@ -14,6 +14,7 @@
 using Microsoft.AspNetCore.Http.Features;
```

Comment:
I think you're also missing an API to make it easy to add a test startup configure services filter. E.g. `IWebHostBuilder.AddTestServices(services => { ... });`